### DISCUSSION TITLE
Hook to change the theme

### DISCUSSION THREAD

**Discussion by AaronAbuUsama** [rainbow-me/rainbowkit#696] [Ideas]
Many apps have light and dark mode. It would be nice to have a hook that changed the theme along with the rest of the site. The only way i can see to do this is to wrap the entire provider and make a custom hook

**Comment by shrugs** [comment]
write a context, put it above rainbow provider, pass the context's state to the rainbow provider as theme variable, change that context from elsewhere in the app
